emcc.py

```diff
@@ -3512,6 +3513,8 @@ def consume_arg_file():
     elif check_arg('--embind-emit-tsd'):
       options.embind_emit_tsd = consume_arg()
       settings.INVOKE_RUN = False
+    elif check_arg('--embind-debug'):
+      settings.EMBIND_DEBUG = consume_arg()
```

Comment:
I don't think we need to new command line flag for this.   Just `-sEMBIND_DEBUG` should be enough.

Reply:
I changed